Makefile

```diff
@@ -442,9 +446,6 @@ e2e-build-local-ratify-image:
 	-t localbuild:test .
 	kind load docker-image --name kind localbuild:test
 
-e2e-uninstall-ratify:
-	./.staging/helm/linux-amd64/helm uninstall ${RATIFY_NAME} --namespace ${GATEKEEPER_NAMESPACE}
```

Comment:
it's added in the previous policy PR. Now we use CRD to enable rego policy instead of using helm uninstall/install, so it's not used anymore. We could add it back in the future if used.